_posts/2016-10-24-deploy-jekyll-with-travis.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: Deploying Jekyll to Github Pages with Travis and Gulp
+date: 2016-10-24
+author: Anne Tomasevich
+tags: jekyll travis
+summary: |
+  A step-by-step guide to how we deploy our gulpified Jekyll site to GitHub
+  Pages using Travis, opening the door for us to use Jekyll Plugins.
+---
+
+Since our company's inception, we at Savas Labs have used [Jekyll](https://jekyllrb.com/) to build our company website and [GitHub Pages](https://pages.github.com/) to host it with great success. Jekyll is a powerful tool out of the box, and it's hard to imagine a simpler hosting strategy than GitHub Pages, which only requires a [specifically named branch](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/) and a [single settings update](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#enabling-github-pages-to-publish-your-site-from-master-or-gh-pages) to get your site live for free.
+
+One well-known (and completely understandable) limitation of this workflow: any Jekyll plugins not [whitelisted by GitHub Pages](https://pages.github.com/versions/) will be disabled when GitHub Pages builds the site, since `jekyll build` is run with the `--safe` tag. Many solutions exist to get around this, and when we decided to start using Jekyll plugins on our site we opted to use [Travis CI](https://travis-ci.org/) to build our site and push the compiled site to our `master` branch, to then be launched via GitHub Pages. To complicate matters, we recently starting building our site with gulp, which introduced another layer of dependencies to the mix.
```

Comment:
I also think it would be helpful to mention here why `we recently starting building our site with gulp` and link to your last post on optimizing performance with Gulp.